db/db_universal_compaction_test.cc

```diff
@@ -2150,6 +2150,72 @@ TEST_F(DBTestUniversalDeleteTrigCompaction, IngestBehind) {
   ASSERT_GT(NumTableFilesAtLevel(5), 0);
 }
 
+TEST_F(DBTestUniversalDeleteTrigCompaction, PeriodicCompaction) {
```

Comment:
I think this test should be under "DBTestUniversalCompaction" instead of "DBTestUniversalDeleteTrigCompaction". 
However, the test code itself looks and it is easy to follow.

Reply:
It feels very odd but non existing test under DBTestUniversalDeleteTrigCompaction is related to delete triggered compaction. I'll just rename this class name to DBTestUniversalCompaction2.